deps/external-gtest.cmake

```diff
@@ -5,33 +5,19 @@
 message( STATUS "Configuring gtest external dependency" )
 include( ExternalProject )
 
-# set( gtest_version "1.7.0" CACHE STRING "gtest version to download/use" )
-# mark_as_advanced( gtest_version )
-
-# If the user does not specify an explicit fortran compiler, assume gfortran
-if( NOT DEFINED CMAKE_C_COMPILER )
-    set( CMAKE_C_COMPILER cc )
-endif( )
+# set( gtest_cmake_args -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>/package )
+set( PREFIX_GTEST ${CMAKE_INSTALL_PREFIX} CACHE PATH "Location where boost should install, defaults to /usr/local" )
```

Comment:
if it defaults to   /usr/local   we need to be sure to tell the user to build with   sudo